### ISSUE TITLE
Adding geometry to attribute data

### ISSUE THREAD

**Issue by hasiber** [opengisch/QField#2019]
There is no way to add a geometry to an object with no geometry data. 

I have a project with a (point)table. The data is from an old survey wich was done only on paper without geometry data.
Now I prepared the data for qfield and wanted to add the geometry in the field. But there is no button for this.
If there is a geometry you could edit it (there is a button). 
I think it would be enough to enable this (edit) button also for objects without geometry data in geometry tables/layers.

**Comment by suricactus** [comment]
Can you add an example project? I may look into that in the future.

**Comment by hasiber** [comment]
[GrenzeBsp.zip](https://github.com/opengisch/QField/files/6704311/GrenzeBsp.zip)

**Comment by lp-dj** [comment]
Had a similar problem. Helped myself by creating a new geometry layer and linking it by relationship to the original (in your case geometry-less) layer. Since version 1.9 you are fortunately able to capture geometry when adding a child object.

Once the field work is done, geometries and attributes could then be easily merged into one layer if needed.

Maybe this helps you too for the time being?
